builders/build/eth-api/dev-env/brownie.md

```diff
@@ -138,7 +128,7 @@ If you would like to set a default network, you can do so by adding the followin
 
     ```yaml
     networks:
-        default: moonbeam-dev
+        default: moonbase
```

Comment:
This as close as I could get, but could you please update `{INSERT_ACCOUNT_NAME}` to be just `INSERT_ACCOUNT_NAME` without the curlys?